core/config/v2/types.go

```diff
@@ -1126,3 +1288,24 @@ func (m *MercurySecrets) ValidateConfig() (err error) {
 type ThresholdKeyShareSecrets struct {
 	ThresholdKeyShare *models.Secret
 }
+
+func (t *ThresholdKeyShareSecrets) SetFrom(f *ThresholdKeyShareSecrets) (err error) {
+	err = t.validateMerge(f)
+	if err != nil {
+		return err
+	}
+
+	if v := f.ThresholdKeyShare; v != nil {
+		t.ThresholdKeyShare = v
+	}
+
+	return nil
+}
+
+func (t *ThresholdKeyShareSecrets) validateMerge(f *ThresholdKeyShareSecrets) (err error) {
+	if t.ThresholdKeyShare != nil && f.ThresholdKeyShare != nil {
+		err = multierr.Append(err, ErrOverride{Name: "ThresholdKeyShare", Msg: "overrides (duplicate keys or list elements) are not allowed for multiple secrets files"})
```

Comment:
Is every usage of `ErrOverride` passing the same`Msg`? If so, can just inline it in the `fmt.Sprintf` instead.
```suggestion
		err = multierr.Append(err, ErrOverride{Name: "ThresholdKeyShare"})
```

Reply:
Thanks, will do! I believe we won't need a dynamic message since the field name itself clearly signals the fix.